smsender/src/main/java/fr/iolabs/leaf/smsender/services/SmsService.java

```diff
@@ -0,0 +1,11 @@
+package fr.iolabs.leaf.smsender.services;
+
+public interface SmsService {
+    /**
+     * Sends sms to a specified number
+     *
+     * @param to phone number of the recipient.
+     * @param message content of the SMS message.
+     */
+    void sendSMS(String to, String message);
```

Comment:
For the to parameter, let's use a LeafAccount instead of a String.
And find the user phone number from the LeafAccount: `account.getProfile().getPhoneNumber()` with all never null check.